src/sdk/lsf-sdk.js

```diff
@@ -133,20 +132,23 @@ export class LSFWrapper {
       interfaces.push("annotations:comments");
     }
 
-    console.group("Interfaces");
-    console.log([...interfaces]);
+    if (this.role) {
+      if (this.role === 'REVIEWER') interfaces.push("annotations:view-all");
+      if ((this.role === 'ADMIN' || this.role === 'MANAGER' || this.role === "OWNER") && !this.labelStream) {
```

Comment:
I think this is exclusive of the role, and therefore should be an `else if`

```suggestion
      else if ((this.role === 'ADMIN' || this.role === 'MANAGER' || this.role === "OWNER") && !this.labelStream) {
```